CHANGELOG.md

```diff
@@ -56,6 +56,7 @@
 	If you are upgrading, and have added/modified TelemetryProcessors, make sure to copy them to the default sink section.
 - [Microsoft.AspNet.TelemetryCorrelaiton package update to 1.0.4](https://github.com/Microsoft/ApplicationInsights-dotnet-server/issues/991)
 - Add vmScaleSetName field to heartbeat properties collected by AzureInstanceMetadataTelemetryModule to allow navigation to right Azure VM Scale Set
+- [Allow users to ignore specific UnobservedTaskExceptions](https://github.com/Microsoft/ApplicationInsights-dotnet-server/issues/1026)
```

Comment:
so the description say it "allows to ignore" when the code simply ignores by defult. Should there be a setting that will enable to restore the original behavior?

Reply:
@SergeyKanzhelev it only ignores exceptions explicitly marked as `Observed` by the user subscribing to the event and calling `SetObserved` for that exception. Other exceptions are treated as before..

Reply:
I was asking - if someone get used to the behavior when Observer exceptions are still being collected, this change will break the expectation. So I am suggesting to add a setting that will keep the previous behavior.

Reply:
But there is one.. the `Observed` setting :)
If they previously observed the exception they probably didn't want another handler, and if they do, they can just not set it as observed anymore (which how the next handler is controlled).
Practically, this is such a niche edge case (even more than this one is) so I would leave that as is unless someone complains..